src/Reanimated.res

```diff
@@ -1,9 +1,340 @@
+  ~backgroundColor: ReactNative.Color.t=?,
+  ~borderBottomColor: ReactNative.Color.t=?,
+  ~borderBottomEndRadius: float=?,
+  ~borderBottomLeftRadius: float=?,
+  ~borderBottomRightRadius: float=?,
+  ~borderBottomStartRadius: float=?,
+  ~borderBottomWidth: float=?,
+  ~borderColor: ReactNative.Color.t=?,
+  ~borderEndColor: ReactNative.Color.t=?,
+  ~borderEndWidth: float=?,
+  ~borderLeftColor: ReactNative.Color.t=?,
+  ~borderLeftWidth: float=?,
+  ~borderRadius: float=?,
+  ~borderRightColor: ReactNative.Color.t=?,
+  ~borderRightWidth: float=?,
+  ~borderStartColor: ReactNative.Color.t=?,
+  ~borderStartWidth: float=?,
+  ~borderStyle: ReanimatedStyle.borderStyle=?,
+  ~borderTopColor: ReactNative.Color.t=?,
+  ~borderTopEndRadius: float=?,
+  ~borderTopLeftRadius: float=?,
+  ~borderTopRightRadius: float=?,
+  ~borderTopStartRadius: float=?,
+  ~borderTopWidth: float=?,
+  ~borderWidth: float=?,
+  ~bottom: ReanimatedStyle.size=?,
+  ~direction: ReanimatedStyle.direction=?,
+  ~display: ReanimatedStyle.display=?,
+  ~easing: float => float=?,
+  ~elevation: float=?,
+  ~end: ReanimatedStyle.size=?,
+  ~flex: float=?,
+  ~flexBasis: ReanimatedStyle.margin=?,
+  ~flexDirection: ReanimatedStyle.flexDirection=?,
+  ~flexGrow: float=?,
+  ~flexShrink: float=?,
+  ~flexWrap: ReanimatedStyle.flexWrap=?,
+  ~height: ReanimatedStyle.size=?,
+  ~justifyContent: ReanimatedStyle.justifyContent=?,
+  ~left: ReanimatedStyle.size=?,
+  ~margin: ReanimatedStyle.margin=?,
+  ~marginBottom: ReanimatedStyle.margin=?,
+  ~marginEnd: ReanimatedStyle.margin=?,
+  ~marginHorizontal: ReanimatedStyle.margin=?,
+  ~marginLeft: ReanimatedStyle.margin=?,
+  ~marginRight: ReanimatedStyle.margin=?,
+  ~marginStart: ReanimatedStyle.margin=?,
+  ~marginTop: ReanimatedStyle.margin=?,
+  ~marginVertical: ReanimatedStyle.margin=?,
+  ~maxHeight: ReanimatedStyle.size=?,
+  ~maxWidth: ReanimatedStyle.size=?,
+  ~minHeight: ReanimatedStyle.size=?,
+  ~minWidth: ReanimatedStyle.size=?,
+  ~opacity: float=?,
+  ~originX: float=?,
+  ~originY: float=?,
+  ~overflow: ReanimatedStyle.overflow=?,
+  ~padding: ReanimatedStyle.size=?,
+  ~paddingBottom: ReanimatedStyle.size=?,
+  ~paddingEnd: ReanimatedStyle.size=?,
+  ~paddingHorizontal: ReanimatedStyle.size=?,
+  ~paddingLeft: ReanimatedStyle.size=?,
+  ~paddingRight: ReanimatedStyle.size=?,
+  ~paddingStart: ReanimatedStyle.size=?,
+  ~paddingTop: ReanimatedStyle.size=?,
+  ~paddingVertical: ReanimatedStyle.size=?,
+  ~position: ReanimatedStyle.position=?,
+  ~right: ReanimatedStyle.size=?,
+  ~shadowColor: ReactNative.Color.t=?,
+  ~shadowOffset: ReanimatedStyle.offset=?,
+  ~shadowOpacity: float=?,
+  ~shadowRadius: float=?,
+  ~start: ReanimatedStyle.size=?,
+  ~top: ReanimatedStyle.size=?,
+  ~transform: array<ReanimatedStyle.transform>=?,
+  ~width: ReanimatedStyle.size=?,
+  ~zIndex: int=?,
+  unit,
+) => reanimatedStyle = ""
+
+module LayoutAnimation = {
+  type t = unit
```

Comment:
Could `t` be opaque? That way we would avoid someone doing something like this:
```rescript
<Reanimated.View entering=() />
```

Reply:
Yes, it should be opaque. Will do this

Reply:
Fixed